docker-compose.core.yml

```diff
@@ -61,7 +61,7 @@ services:
     extends:
       file: docker-compose.services.yml
       service: app
-    command: poetry run watchmedo auto-restart -d . -p '*.py' -- celery worker -E -B -A inspirehep.celery -l INFO
+    command: poetry run watchmedo auto-restart -d . -p '*.py' -- celery worker -E -A inspirehep.celery -l INFO --purge --queues celery,migrator,indexer_task
```

Comment:
names of the queues are the same as for legacy. Does anything need to be done to be sure the inspire-next workers don't comsume these queues any more?

Reply:
we'll remove the queues from ``inspire-next`` workers when we switch migrator and indexer in inspirehep.